Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/ExchangeManagerBean.java

```diff
@@ -161,9 +169,7 @@ public void setFormExchangeEnable(boolean enable) {
 
     // datatable-list
     public List<ExchangeType> getExchanges() {
-        if (null == exchanges) {
-            exchanges = exchangeService.getAllExchanges();
-        }
+        exchanges = exchangeService.getAllExchanges();
         return exchanges;
```

Comment:
we should return exchangeService.getAllExchanges(); instead of assigned and return